docs/server-side/v2/api-reference/analytics/recording-events.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Recording Events
+nav: 3.63
+---
+
+Analytics APIs can be used to fetch the recording events similar to [webhook](/server-side/v2/how--to-guides/configure-webhooks/webhook#beamrecordingsuccess)
+Currently there is support for `beam.recording.success` event. This event is sent when beam successfully records the room and uploads the video to storage.
+
+<GetRequest title="https://api.100ms.live/v2/analytics/events" />
+<Request id="req-0">
+
+```bash
+curl --location --request GET 'https://api.100ms.live/v2/analytics/events' \
```

Comment:
noticed both pages have the same endpoint. is that correct?

Reply:
yes, they have the same endpoint, events are filtered by the type query parameter. right now we have made it mandatory. it can be reveiwed again later to get the mix of all events.

Reply:
got it. i think it'll be useful to call out the mandatory params in this snippet - so that they are clearly distinct